modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -43,7 +43,7 @@ printf "BACKUP COMPLETED IN: ${TIME}s\n"
 
 # Compress and encrypt backup
 TIME="$(date +%s)"
-(/bin/tar cvz $BACKUP_DIR | /usr/bin/gpg -e -o $BACKUP_DIR/$BACKUP_FILE -r $KEY_FINGERPRINT)
+(/bin/tar cvz $BACKUP_DIR --exclude=/var/lib/s3backup/lost+found | /usr/bin/gpg -e -o $BACKUP_DIR/$BACKUP_FILE -r $KEY_FINGERPRINT)
```

Comment:
This can probably just be "lost+found" as the exclude just matches expressions, and it removes the hard coded directory location